conntrollers/controller.js

```diff
@@ -0,0 +1,11 @@
+class Controller {
+
+    sendHelloMessage(req, res) {
+        res.json({
+            message: 'Hello my dear friend!'
+        })
+    }
+
+}
+
+module.exports = new Controller()
```

Comment:
As you can see there is an icon with a `No newline at the end of file` tooltip message. You can avoid this if you configure your IDE in a proper way (or after adding prettier you can set it as your IDE's default formatter (and configure it to format files on save) and add a simple script (or maybe a hook on commit) to format your code)

Reply:
ok